spec/controllers/welcome_controller_spec.rb

```diff
@@ -0,0 +1,9 @@
+require "rails_helper"
+
+RSpec.describe WelcomeController do
+  context "get index" do
+    it "succesfully show home page" do
+      get :index
```

Comment:
1. Let's move our action call to subject at the top

```ruby
subject(:perform_request) { get :index }
```

2. We miss some assertion. Currently you just call the `index` action and check nothing.

```ruby
before { perform_request }

it 'returns success response' do
   expect(response).to be_success
end
```